Godeps/Godeps.json

```diff
@@ -53,6 +53,10 @@
 		{
 			"ImportPath": "github.com/syndtr/gocapability/capability",
 			"Rev": "e55e5833692b49e49a0073ad5baf7803f21bebf4"
+		},
```

Comment:
Fixed a merge conflict by hand, and didn't verify it still worked afterwards... Oops.